integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -719,8 +718,7 @@ void shouldCallAuthorizeAsDocAppClient(String uiLocales) throws JOSEException, P
                 authRequest.getCustomParameter("vtr"),
                 equalTo(List.of("[\"P2.Cl.Cm\",\"Cl.Cm\"]")));
```

Comment:
This test includes an invalid vtr list (mix of identity and non-identity) - this should be caught with the updated logic in ato-248